x/permission/types/keys.go

```diff
@@ -82,13 +82,22 @@ func GetPolicyByIDKey(policyID math.Uint) []byte {
 }
 
 func GroupMembersPrefix(groupID math.Uint) []byte {
-	return append(GroupMemberPrefix, groupID.Bytes()...)
+	return append(GroupMemberPrefix, LengthPrefix(groupID.Bytes())...)
 }
 
 func GetGroupMemberKey(groupID math.Uint, member sdk.AccAddress) []byte {
-	return append(GroupMemberPrefix, append(groupID.Bytes(), member.Bytes()...)...)
+	return append(GroupMemberPrefix, append(LengthPrefix(groupID.Bytes()), member.Bytes()...)...)
 }
 
 func GetGroupMemberByIDKey(memberID math.Uint) []byte {
 	return append(GroupMemberByIDPrefix, memberID.Bytes()...)
 }
+
+func LengthPrefix(bz []byte) []byte {
```

Comment:
I think it makes more sense to use a fixed length?

Reply:
```go
func FixedUintBytes(n math.Uint) []byte {
	buf := make([]byte, math.MaxBitLen)

	return n.BigInt().FillBytes(buf)
}
```

Reply:
good idea. Refactored.